src/content/docs/infrastructure/amazon-integrations/manage-aws-data/aws-integrations-metrics.mdx

```diff
@@ -11,68 +11,89 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-We recommend using the [AWS CloudWatch Metric Streams integration](/docs/integrations/amazon-integrations/aws-integrations-list/aws-metric-stream/) to ingest AWS services' metrics. For a complete list of available metrics, check the CloudWatch metrics listing documentation for each AWS service.
+Once you're collecting AWS data, you can set up alert conditions or query your data in the New Relic platform. This doc describes how we store AWS dimensional metrics so you can locate AWS namespace entities and create alerts and queries about the data that matters to you. This doc then provides you with example queries and alerts to get you started monitoring your AWS environment. 
```

Comment:
"Once you're collecting" seems like a weird start to me for some reason, just reads oddly.

The last sentence should probably be something like "This doc also provides you with" instead of "This doc then provides you with"

Reply:
Good catch! this is a wordy paragraph so will do some light rewrites